faq/faq.rst

```diff
@@ -303,6 +303,44 @@ Q: Which API’s have been implemented in OP-TEE?
 
     - `Android Verified Boot 2.0`_ (AVB 2.0)
 
+Q: Which Linux kernel version supports <some OP-TEE feature>?
+=============================================================
+    - The OP-TEE Linux driver is maintained in the official Linux tree at
+      `kernel.org`_ under `drivers/tee`. This is normally where you find the
+      latest code. That being said, some platforms need minor customizations,
+      such as device tree updates, in order to be used in the OP-TEE developer
+      builds (:ref:`manifest` files). That is why the `linaro-swg kernel`_
+      branch ``optee`` is used in the manifest files. It is rebased onto
+      upstream on a regular basis.
+    - Older kernels may lack support for newer OP-TEE features. In order to
+      assess in which kernel version some commit has been introduced, you may
+      use the following shell command:
+
+      .. code-block:: bash
+
+          $ cd linux
+          $ git log --no-merges --oneline v5.12 drivers/tee | \
```

Comment:
You may want to put `--` between the tag and the path.

Reply:
For me `5.12` it's obvious that it's an example. But I wouldn't be surprised if someone read that verbatim in the future.

Reply:
@jbech-linaro right, I should probably just remove the tag so that the history of the current branch is printed, and that would address Jens' comment too.